gym/core.py

```diff
@@ -26,9 +26,8 @@ class Env(object):
 
     Note: a default reward range set to [-inf,+inf] already exists. Set it if you want a narrower range.
 
-    The methods are accessed publicly as "step", "reset", etc.. The
-    non-underscored versions are wrapper methods to which we may add
-    functionality over time.
+    The methods are accessed publicly as "step", "reset", etc...
+    The non-underscored versions are wrapper methods to which we may add functionality over time.
```

Comment:
the part about non-underscored methods is obsolete and can be removed.

Reply:
done